src/content/tutorials/access-management/iam-roles-for-service-accounts/index.md

```diff
@@ -0,0 +1,123 @@
+---
+linkTitle: IAM roles for service accounts
+title: AWS IAM roles for service accounts (IRSA)
+description: This article describes how to use a new feature that allows binding of specific AWS IAM roles to a service account of a pod.
+weight: 60
+menu:
+  principal:
+    parent: tutorials-access-management
+    identifier: tutorials-access-management-irsa
+user_questions:
+ -  How can I use IAM roles for service accounts?
+last_review_date: 2024-12-03
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+*Basic knowledge about how service accounts work in Kubernetes is necessary for understanding what this page is about. If you could use a quick refresher, we strongly suggest taking a look at [this article](https://learnk8s.io/authentication-kubernetes).*
+
```

Comment:
```suggestion
```

I don't think we should add so many details to a tutorial. The page already clearly explains that service accounts must be used.